doc/look_and_feel.md

```diff
@@ -2,6 +2,20 @@
 
 At this point you're wondering how you can customize the appearance of applications in newm.
 Here's some help.
+
+## Window decorations
+
+By default, `pywm` encourages clients to draw their own decorations.
+If you prefer no decorations, then you can tell clients otherwise:
```

Comment:
Well, https://github.com/jbuchermn/newm/blob/master/doc/config.md#config-general-appearance does document "A very basic server-side decoration implementation is available (unicolor rounded corners border around a view)." and flags to control it.

But essentially yes, if the point of decorations is show title, current CSD is effectively ~no decorations :-)

It's also worth mentioning (here and/or there) that this settings only affects new windows.  If you change it and reload the config, existing clients already negotiated either CSD or SSD and are not going to change.

Reply:
Whoops, I realize I skipped past that first sentence. I saw "look_and_feel.md" and I expected to see newm settings. I'm going to remove this, and suggest instead:
```diff
  At this point you're wondering how you can customize the appearance of applications in newm.
  Here's some help.
+ If you want to customize the appearance of newm itself,
+ [see the corresponding section in newm](doc/config.md#config-general-appearance](./doc/config.md#config-general-appearance)
```